programs/protocol_event/src/instructions/create_grouping.rs

```diff
@@ -0,0 +1,170 @@
+use crate::error::EventError;
+use crate::state::category::Category;
+use crate::state::event_group::EventGroup;
+use anchor_lang::prelude::*;
+
+pub fn create_category(
+    category: &mut Category,
+    code: String,
+    name: String,
+    payer: Pubkey,
```

Comment:
NP The pedant in me might try to align the order of arguments between `create_category` and `create_event_group` given the share 3 of them :twitch: